build.sbt

```diff
@@ -186,7 +196,7 @@ lazy val flywaySettings = Seq(
 lazy val gem = project
   .in(file("."))
   .settings(commonSettings)
-  .aggregate(coreJVM, db, json, ocs2, ephemeris, service, telnetd, ctl, web, sql, main)
+  .aggregate(coreJVM, coreJS, db, json, ocs2, ephemeris, service, telnetd, ctl, web, sql, main, ui)
```

Comment:
Size aside, how much does this add to compile time?

Reply:
Travis shows maybe 1m more though it varies, in some travis runs is even faster but that's likely due to the way travis work. This number will increase though as we evolve the ui especially `fullOptJS`